content/en/docs/customizing-objects/expressions.md

```diff
@@ -62,6 +62,42 @@ public class JavaClass {
 "objectList[0].nestedField"
 ```
 
+### Selecting Properties Using JavaGetter Selector
+
+There is a type-safe way to select properties using a `javaGetter()` property selector.
+This selector is designed to choose and represent a property through a getter method reference in Java.
+
+##### Selecting the root object:
+- Currently Not Supported
+
+##### Selecting a specific field:
+```java
+javaGetter(JavaClass::getField)
+```
+
+##### Selecting a nested field:
+```java
+javaGetter(JavaClass::getObject).into(Nested::getNestedField)
+```
+
+##### Selecting the n-th element of a collection:
+```java
+javaGetter(JavaClass::getList).index(String.class, n)
+```
+
+##### Selecting all elements of a collection:
+```java
+javaGetter(JavaClass::getList).allIndex(String.class)
+```
```

Comment:
It is better to add examples of array.

Reply:
I also found this comment unclear. Can you give me an example?

Reply:
```java
javaGetter(JavaClass::getArray).allIndex(String.class)
```